src/components/ClassroomComponent/ProfileComponent/ProfileContainer.js

```diff
@@ -187,15 +187,14 @@ class ProfileContainer extends Component {
                 floatingLabelText='Name Abbreviation'
                 type='text'
                 value={this.state.tempUser && this.state.tempUser.abbrv}
```

Comment:
The `tempUser` object needs some change too. We could simply use `this.props.user.abbrv` like in email